### PR TITLE
New benchmarkWriteSubRead

### PR DESCRIPTION
**Pull request by aboodman**

### CODE REVIEW

File: perf/replicache.ts [hunk 1]
```diff
@@ -155,193 +152,107 @@ export function benchmarkCreateIndex(opts: {
...
-  const maxCount = 1000;
-  const minCount = 10;
+  const {valueSize, numSubsTotal, keysPerSub, keysWatchedPerSub, numSubsDirty} =
+    opts;
+
+  const numKeys = keysPerSub * numSubsTotal;
+  const cacheSizeMB = (numKeys * valueSize) / 1024 / 1024;
+  const kbReadPerSub = (keysWatchedPerSub * valueSize) / 1024;
+
+  const data = Array.from({length: numKeys}).map(() => randomString(valueSize));
```
Comment (arv) [inline perf/replicache.ts:177]: Can we change this to randomJSON?

**Review by arv — COMMENTED** [review]
LGTM

Thanks

File: perf/replicache.ts (hunk 1 above)
Comment (aboodman) [reply perf/replicache.ts:177]: I did that in a subsequent commit. It didn't make a huge difference with a realistic number of keys.

Comment (arv) [reply perf/replicache.ts:177]: I have some ideas for improving hashing so I think that it would be useful to have more realistic data.

**Pull request merged**